docs/releases/upgrading/upgrade-v12.0.md

```diff
@@ -0,0 +1,164 @@
+# Upgrading from v11 to v12
+
+## Prerequisites
+
+The steps listed in this article require an existing local installation of InvenioRDM v11.
+
+!!! warning "Backup"
+
+    Always backup your database and files before you try to perform an upgrade.
+
+!!! info "Older Versions"
+
+    In case you have an InvenioRDM installation older than v11, you can gradually upgrade
+    using the existing infrastructure to v11 and afterwards continue from here.
```

Comment:
```suggestion
    using the existing infrastructure to v11 and afterward continue from here.
```

Reply:
depends on whether we're generally using American (afterward) or British (afterwards) English